runner_helper.rb

```diff
@@ -0,0 +1,9 @@
+require "CSV"
```

Comment:
Should we also go ahead and add our SimpleCov requirements at the beginning of this helper?

`require 'simplecov'
SimpleCov.start`